### PR TITLE
Crosshairs

### PR DESCRIPTION
**Pull request by bjkomer**
Adding crosshairs to value plots. Aimed to make it similar to how they worked in Nengo 1.4. Still needs some cleanup work, but I want to get a sense of how people feel about it so far. Addresses #532

### CODE REVIEW

**Comment by tcstewar** [comment]
Nice!  I haven't looked at the code yet, but visually that looks great!

File: nengo_gui/static/components/value.js [hunk 1]
```diff
@@ -42,6 +42,64 @@ Nengo.Value = function(parent, sim, args) {
...
+	    .on('mouseover', function() {
+		var mouse = d3.mouse(this);
+		self.crosshair_updates = true;
+                self.crosshair_g.style('display', null);
+		self.cross_hair_mouse = [mouse[0], mouse[1]]
+	    })
+            .on('mouseout', function() {
+		var mouse = d3.mouse(this);
+		self.crosshair_updates = false;
+                self.crosshair_g.style('display', 'none');
```
Comment (Seanny123) [inline nengo_gui/static/components/value.js]: Indentation

File: nengo_gui/static/components/value.js [hunk 2]
```diff
@@ -42,6 +42,64 @@ Nengo.Value = function(parent, sim, args) {
...
+		self.cross_hair_mouse = [mouse[0], mouse[1]]
+	    })
+	    .on('mousemove', function() {
+		var mouse = d3.mouse(this);
+		self.crosshair_updates = true;
+		self.cross_hair_mouse = [mouse[0], mouse[1]]
```
Comment (Seanny123) [inline nengo_gui/static/components/value.js]: Indentation and semicolon

File: nengo_gui/static/components/value.js [hunk 3]
```diff
@@ -56,6 +114,46 @@ Nengo.Value = function(parent, sim, args) {
...
+	      });
+	    
+	    this.crosshair_g.select('#crosshairYtext')
+	      .attr('x', this.axes2d.ax_right)
+	      .attr('y', y-2)
+	      .text(function () {
+		      return Math.round(self.axes2d.scale_y.invert(y) * 100) / 100;
+	      });
+    }
+    else {
```
Comment (Seanny123) [inline nengo_gui/static/components/value.js]: `else` is supposed to be cuddled by two curly brackets

**Comment by Seanny123** [comment]
A few style issues (because we haven't included a syntax checker in our workflow yet), but other than that it looks good to me.

**Comment by Seanny123** [comment]
@bjkomer btw, this needs to be rebased now that my monster branch for Spa Similarity has been merged.

**Pull request merged**